WIP - Joesandbox (#2124)
rapid7/insightconnect-plugins: plugins/joe_sandbox/help.md

|Most Relevant Analysis|object|None|None|Webid, detection & score|None|
|name|string|None|None|name|None|
|Status|string|None|None|Status of the analysis e.g. submitted, running, finished|None|
|Submission ID|string|None|None|submission ID|None|
|time|string|None|None|time|None|
  
**analysis**

|Name|Type|Default|Required|Description|Example|
| :--- | :--- | :--- | :--- | :--- | :--- |
|AnalysisID|string|None|None|Analysis ID. Will not be returned if the analysis is not finished|None|
|Comments|string|None|None|Comments|None|
|Duration|integer|None|None|Duration of the analysis in seconds (only for finished analyses)|None|
|Filename|string|None|None|File name|None|
|MD5|string|None|None|MD5|None|
|Runs|[]run|None|None|Runs|None|
|Scriptname|string|None|None|Script name|None|
|SHA1|string|None|None|SHA1|None|
|SHA256|string|None|None|SHA256|None|
|Status|string|None|None|Status of the analysis, one of: submitted, running, finished|None|
|Tags|[]string|None|None|Tags|None|
|Time|date|None|None|Time|None|
|WebID|string|None|None|Web ID|None|

## Troubleshooting
  
*There is no troubleshooting for this plugin.*

# Version History
  

> * 1.0.0 - Initial plugin  
> * 2.0.0 updated joe_sandbox libray from 2.9.5 to 3.21.0  - updated actions from breaking changes in API. 

Reviewer: ```suggestion
* 2.0.0 - Updated `jbxapi` dependency from 2.9.5 to 3.21.0 | Updated (list action names) from breaking changes in API. 
* 1.0.4 - Add extra optional input for Submit Sample action
* 1.0.3 - Add example inputs
* 1.0.2 - Fix misspelling in error message | Remove generic "automation" keyword
* 1.0.1 - New spec and help.md format for the Extension Library
* 1.0.0 - Initial plugin  
```

> https://github.com/joesecurity/jbxapi/blob/master/CHANGES.md#version-300

Reviewer: ```suggestion
```

# Links

> 

Reviewer: ```suggestion
* [Joe Sandbox](https://www.joesecurity.org)
```

## References
  

> * https://www.joesecurity.org

Reviewer: ```suggestion
* [Joe Sandbox API](https://jbxcloud.joesecurity.org/userguide?sphinxurl=usage%2Fwebapi.html)
* [Joe Sandbox API wrapper](https://github.com/joesecurity/jbxapi)
* [Report formats](https://jbxcloud.joesecurity.org/userguide?sphinxurl=usage/reportformats.html)
```
